test/images/README.md

```diff
@@ -105,6 +112,12 @@ limitations when it comes to the Windows images:
   (make sure that process isolation is used, not Hyper-V isolation).
   [This](https://stefanscherer.github.io/find-dependencies-in-windows-containers/) is a useful guide on how to use `procmon.exe`.
 
+For Windows, in order to spawn process-isolated containers, the container OS version should closely match
+the host OS version. For this reason, we build test images for different Windows OS Versions: 1809, 1903,
+1909. In order add support for a new Windows OS version, a new entry for that OS version will have to
+be first added to the `windows-servercore-cache` and `busybox` images, followed by the rest of the images.
```

Comment:
Can we provide a link to test/images/windows/README.md here?

Reply:
The doc talks about how to build your own.  Where do the official ones live and how do publish for use here?  Can we add a section for that to the readme below?

Reply:
For ``windows-servercore-cache`` and ``busybox`` images, this same README file applies. They are built the same way all other regular test image is built. Don't mistake them for the ``busybox-helper`` and ``powershell-helper`` images.

Reply:
As for the second question, I'm not sure I understand what you're referring to. Do you mean the Image Promotion process / official E2E test registry? If so, it's already been covered in the other sections of this README.

Reply:
I think I got them confused with `busybox-helper` and `powershell-helper`.  Are there docs on how those are built and published?

Reply:
> I think I got them confused with busybox-helper and powershell-helper. Are there docs on how those are built and published?

This PR adds a readme to /test/images/windows/ which is where the helper images get built.